ts/session/apis/open_group_api/sogsv3/sogsV3BatchPoll.ts

```diff
@@ -285,9 +285,11 @@ const makeBatchRequestPayload = (
         json: {
           rooms: [options.addRemoveModerators.roomId],
           global: false,
-          // moderator: isAddMod, // currently we only support adding/removing visible admins
           visible: true,
           admin: isAddMod,
+          // currently we only support adding/removing visible admins but we still need to set the `moderator`
+          // permissions here so removing an admin works does not only devote an admin to a moderator
```

Comment:
This comment is a bit confusing. Do you mean we can remove admins and not moderators and so we need separate variables for both so that when we remove an admin we don't remove the moderator as well by mistake?